src/lib.rs

```diff
@@ -149,13 +199,15 @@ impl Context {
 
         Ok(Self {
             context_impl: imple,
+            _marker: PhantomData,
         })
     }
 }
 
 pub struct Surface {
```

Comment:
We need struct-level rustdoc

Reply:
Plus a root-level `#![warn(missing_docs)]` so that it doesn't get missed again